deploy/helm/jumpstarter/charts/jumpstarter-controller/templates/controller-ingress.yaml

```diff
@@ -23,14 +23,12 @@ spec:
             name: jumpstarter-grpc
             port:
               number: 8082
-  {{ if .Values.grpc.tls.enabled }}
   tls:
-  - secretName: {{ .Values.grpc.tls.secret }}
+  - secretName: {{ .Values.grpc.tls.controllerCertSecret }}
```

Comment:
Should this be optional when doing passthrough?

Reply:
Yes, there is really no direct support for pass through in ingress as far as I understood, but I will do it optional